### DISCUSSION TITLE
WindowBuilder empty palette Java

### DISCUSSION THREAD

**Discussion by maryoga** [eclipse/windowbuilder#342] [Q&A] [answered]
Hi Friends,

I have the problems with "empty palette"

I have the next Eclipse version installed

Eclipse IDE for Java Developers (includes Incubating components)
Version: 2022-09 (4.25.0)
Build id: 20220908-1902
OS: Windows 10, v.10.0, x86_64 / win32
Java vendor: Oracle Corporation
Java runtime version: 18.0.1.1+2-6
Java version: 18.0.1.1

![2022-10-15 Eclipse para Desarrolladores Version 2022-09 4 25 0](https://user-images.githubusercontent.com/5562432/196277373-71cbe75f-b9bd-4078-88cc-fa8073748f7d.jpg)

**Comment by wimjongman** [comment]
Why is it a problem? I have never used this. Please open the folders in the pallette.

**Comment by maryoga** [comment]
The problem is that I cannot add a custom component, for example the JCalendar. I can't access the menu as shown in the following image. I cannot access any context menu option

![Vincular JCalendar en Eclipse](https://user-images.githubusercontent.com/5562432/196527788-27ed8c95-be70-4fcd-b374-73ee191f951e.jpg)

**Comment by maryoga** [comment]
Hi wimjongman,

thanks foy you help.

I installed a different Eclipse version which is from 2018 and it worked for me, I don't understand why this happens, it even worked for me with the latest JRE version 1.8 Installed. I don't know how I could have solved this with the version I posted you at the beginning and with the one that gives me that error of not being able to add components.

Eclipse SDK
Version: 2018-09 (4.9)
Build id: I20180906-0745
OS: Windows 10, v.10.0, x86_64 / win32
Java version: 1.8.0_341

![Eclipse versión 2018-09 - 4 9](https://user-images.githubusercontent.com/5562432/196556997-f22b7665-3c96-4488-89cd-8b093d34d82c.jpg)

**Reply by wimjongman** [reply]
I see. I always first add the jar to my classpath and then do "choose component"

![image](https://user-images.githubusercontent.com/180969/197390690-8f54ab85-a711-4510-a7e3-c9f7994c1966.png)

**Comment by wimjongman** [comment, marked as answer]
Please update WB to the latest release 1.11.0. This issue has been resolved.
